src/ast/mod.rs

```diff
@@ -3032,6 +3068,14 @@ pub enum Statement {
         partition: Option<Box<Expr>>,
     },
     /// ```sql
+    /// OPEN cursor_name
+    /// ```
+    /// Opens a cursor.
+    Open {
```

Comment:
I placed this next to `CLOSE` because they're semantically paired, rather than alphabetical. Not sure what's preferred on this project